src/main/java/com/google/sps/servlets/UserActionServlet.java

```diff
@@ -14,35 +14,37 @@
 
 package com.google.sps.servlets;
 
-import com.google.appengine.api.users.UserService;
-import com.google.appengine.api.users.UserServiceFactory;
 import com.google.gson.Gson;
-import com.google.gson.JsonObject;
-import com.google.sps.util.UserHelper;
+import com.google.sps.data.User;
 import java.io.IOException;
-import java.io.PrintWriter;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/**
+ * provides get method for adding a recipe ID to one of the three User recipe lists.
+ * request parameters: "destination", "inputID"
+ * "destination" may be one of three user recipe list: "cookbook", "userRecipes", "planner".
+ * "inputID" must be the recipe ID to be added.
+ */
+@WebServlet("/user-action")
+public class UserActionServlet extends HttpServlet {
 
-@WebServlet("/user-status")
-public class UserStatusServlet extends HttpServlet {
-
-  // Response: returns some user information JSON from the current user's sign-in API profile.
-  // Data: "isUserLoggedIn" : boolean
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("application/json");
+    String destination = (String) request.getParameter("destination");
+    long ID = Long.parse((String) request.getParameter("inputID"));
```

Comment:
nit: lowercase id

Reply:
on that note, which would you say is more correct: fooID or fooId?

Reply:
I think fooId is more in line with the style guide, since we usually camel case abbreviations, for example: someUrl, someGps, etc